scripts/rpm/post-install.sh

```diff
@@ -24,6 +24,22 @@ function install_chkconfig {
     chkconfig --add telegraf
 }
 
+function install_systemd_or_chkonfig {
```

Comment:
Thanks!  But it looks like there's a typo here and also in the disable function 
```suggestion
function install_systemd_or_chkconfig {
```

Reply:
Sorry I didn't notice the mistake )